server/views/pages/product.njk

```diff
@@ -74,9 +74,14 @@
           <li><a class="govuk-link--no-visited-state" href="/monitor/product/{{  product.name | toMonitorName }}">
              Health monitor
           </a></li>
-          <li><a class="govuk-link--no-visited-state" href="/drift-radiator/products/{{ product.name | toMonitorName }}">
+          <li><a class="govuk-link--no-visited-state" href="/drift-radiator/products/{{ product.name }}">
```

Comment:
This one needs to stay, it's just the Slack channel ID is not a monitor name so doesn't need to be piped into that function to clean it up 🙂

Reply:
reverted back